### ISSUE TITLE
List of colors will cause show_all to fail

### ISSUE THREAD

**Issue by MatthiasJ1** [bernhard-42/vscode-ocp-cad-viewer#45]
```py
from build123d import *
from ocp_vscode import *

p = Box(1,1,1)
colors = [Color("black"), Color("white")]

show_all()
```

**Comment by bernhard-42** [comment]
It is a bug, current workarounds:
- `show_all(exclude=["colors"])`
- Omit `Color`, ocp_vscode doesn't need it
  ```python
  b = Box(1,2,3)
  c = Pos(3,0,0) * Cylinder(1,1)
  colors = ["red", (0,255, 0)]
  show_all()
  
  # %%
  show(b,c, colors=colors)
  ```

**Comment by bernhard-42** [comment]
Fixed it upstream in ocp-tessellate.
Will come with the next release

**Comment by bernhard-42** [comment]
fixed in 2.2.2
